### PR TITLE
Added compatibility with php8.1

### PR DESCRIPTION
**Pull request by gusiushub**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
 	],
 	"require": {
 		"yiisoft/yii2": "~2.0.0",
-        "psr/log": "~1.0.2|~1.1.0"
+        "psr/log": "~3.0.0"
```
Comment (ilyaplot) [inline composer.json]: You shouldn't brake backward compatibility.

Comment (gusiushub) [reply composer.json]: Corrected

**Comment by samdark** [comment]
Please fix tests.

**Review by samdark — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -5,5 +5,6 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 8.1
```
Comment (samdark) [inline .travis.yml]: Travis should be removed and 8.1 should be added at https://github.com/samdark/yii2-psr-log-target/blob/master/.github/workflows/build.yml instead.

Comment (gusiushub) [reply .travis.yml]: Corected

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks!

**Pull request merged**